dev-libs/capstone/files/capstone-5.0_rc2-oob-mem-access.patch

```diff
@@ -0,0 +1,37 @@
+Author: Mario Haustein <mario.haustein@hrz.tu-chemnitz.de>
```

Comment:
Link to upstream commit as well? I know but link is in there but hash is missing, so URL kills two birds with one stone

Reply:
I had to rebase the patch due to conflicting commits between the last release an the default branch. So the commit ID wouldn't be found in the official repo. How to solve this?

1. Use the upstream commit ID but with a different patch content?
2. Use the cherry-picked commit ID not part of the upstream repo?

Reply:
Added `Upstream:` tag to the patch for the default branch. Hope that's the right link.